### ISSUE TITLE
False positive: Parameter/variable is overridden, but is never used or appears outside of the scope.

### ISSUE THREAD

**Issue by voku** [kalessil/phpinspectionsea#1184]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Positive                      |
| Plugin         | Php Inspections (EA Ultimate), v2.0.12 |
| Language level | PHP 7.0                        |

Current behaviour (description:)
--
See the error "Parameter/variable is overridden, but is never used or appears outside of the scope."

Expected behaviour
--
No error.

Environment details
---
PhpStorm 2018.3.3
Build #PS-183.5153.36, built on January 9, 2019
Licensed to Portable UTF-8 /  
Subscription is active until April 5, 2019
For non-commercial open source development only.
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.4.0-141-generic

example:
https://3v4l.org/rqmh5

```php
<?php

// init
$resultsTmp = [1, 2, 3];

$list = [];
foreach ($resultsTmp as $resultTmp) {
    $list[] = $resultTmp; // <---
}

$results = &$list;

var_dump($results);
```

**Comment by kalessil** [comment]
Fixed!
